staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -888,8 +889,19 @@ func (c *Cacher) GetList(ctx context.Context, key string, opts storage.ListOptio
 
 	objs, readResourceVersion, indexUsed, err := c.listItems(ctx, listRV, preparedKey, pred, recursive)
 	if err != nil {
+		if consistentRead {
+			if storage.IsTooLargeResourceVersion(err) {
+				metrics.ConsistentReadTotal.WithLabelValues(c.resourcePrefix, "fallback").Add(1)
+				return c.storage.GetList(ctx, key, opts, listObj)
```

Comment:
Reworking my previous example on this updated code.  We have these requests

1.  consistent read fails on something other than istoolarge: I think this `ConsistentReadTotal` labelled with `failure`.
2. consistent read success: I think this is `ConsistentReadTotal` labeled with `success`
3. consistent read fail on istoolarge, backup list succeeds: I think this is `ConsistentReadTotal` labeled with `fallback`
4. consistent read fail on istoolarge, backup list fails: I think this is `ConsistentReadTotal` labeled with `fallback`

Answers I'd like to be able to get from metrics:

1. there have been four total consistent read requests. Correct.
2. 25% of consistent reads succeed. Correct.
3. 50% of backup lists have succeeded. I think currently there is no way to determine.